app/controllers/api/v1/ratings_controller.rb

```diff
@@ -11,34 +11,27 @@ def show
   end
 
   def create
-    users_rated_skills = current_user.ratings.map { |r| r.skill }
-    skill = Skill.where(id: params["rating"]["skill"]).first
-    if users_rated_skills.include?(skill)
-      return revise(skill)
-    end
     rating = Rating.new(rating_params)
-    rating.user = User.where(id: session[:user_id]).first
-    rating.skill = Skill.where(id: params["rating"]["skill"]).first
+    rating.user = current_user
+    rating.skill = Skill.find(params["rating"]["skill"])
```

Comment:
So much better :+1: 

Can we let `skill` or `skill_id` in through the strong params and do something like:

`rating = current_user.ratings.new(rating_params)`